contrib/completion/bash/netctl

```diff
@@ -0,0 +1,463 @@
+#! /bin/bash
+
+GLOBAL_COMMANDS="\
+    group\
+    network\
+    tenant\
+    rule\
+    policy\
+    help"
+
+GLOBAL_OPTIONS="\
+    --netmaster\
+    --help\
+    --version"
+
+# Fetch all tenant names
+_netctl_complete_tenants() {
+    fields='$1'
+    COMPREPLY=( $(compgen -W '$(netctl tenant ls | awk "NR>2 {print $fields}")' -- "$cur") )
```

Comment:
@vijayvikrant - I thought you were going to add `-q` for various objects to avoid having to do awk, etc. i.e. `netctl tenant ls -q` would list the tenants which can then be used not only by you here, but also by the end user to automate things to enlist all object names to iterate upon.

Same comment for various other things.